frameworks/cassandra/README.md

```diff
@@ -1 +1,646 @@
+
+<a name="connecting-clients-to-endpoints"></a>
+## Connecting Clients to Endpoints
+
+To connect to a DC/OS Apache Cassandra cluster using `cqlsh`, first SSH into a host in your DC/OS cluster:
+```
+dcos node ssh --leader --master-proxy
+```
+
+Then, use the `cassandra` Docker image to run `cqlsh`, passing as an argument the address of one of the Apache Cassandra nodes in the cluster:
+```
+docker run cassandra:3.0.10 cqlsh node-0-server.cassandra.mesos
+```
+
+This will open an interactive shell from which you can issue queries and write to the cluster. To ensure that the `cqlsh` client and your cluster are using the same CQL version, be sure to use the version of the `cassandra` Docker image that corresponds to the version of Apache Cassandra being run in your cluster. The version installed by the DC/OS Apache Cassandra Service is 3.0.10.
+
+<a name="managing"></a>
+# Managing
+
+<a name="updating-configuration"></a>
+## Updating Configuration
+
+Edit the runtime environment of the scheduler to make configuration changes. After making a change, the scheduler will be restarted, and it will automatically deploy any detected changes to the service, one node at a time. For example, a given change will first be applied to the `node-0` pod, then `node-1`, and so on.
+
+Nodes are configured with a "Readiness check" to ensure that the underlying service appears to be in a healthy state before continuing with applying a given change to the next node in the sequence. However, this basic check is not foolproof and reasonable care should be taken to ensure that a given configuration change will not negatively affect the behavior of the service.
+
+Some changes, such as decreasing the number of nodes or changing volume requirements, are not supported after initial deployment. See [Limitations](#limitations).
+
+To see a full listing of available options, run `dcos package describe --config cassandra` in the CLI, or browse the DC/OS Apache Cassandra Service install dialog in the DC/OS Dashboard.
+
+<a name="adding-a-node"></a>
+### Adding a Node
+
+The service deploys 3 nodes by default. This may be customized at initial deployment or after the cluster is already running via the `NODES` environment variable. Shrinking the cluster is not supported. If you decrease this value, the scheduler will complain about the configuration change until it's reverted back to its original value or a larger one.
+
+<a name="resizing-a-node"></a>
+### Resizing a Node
+
+The CPU and Memory requirements of each node may be increased or decreased as follows:
+- CPU (1.0 = 1 core): `CASSANDRA_CPUS`
+- Memory (in MB): `CASSANDRA_MEMORY_MB`. To prevent out of memory errors, you must ensure that the `TASKCFG_ALL_CASSANDRA_HEAP_SIZE` environment variable is less than `$CASSANDRA_MEMORY_MB`.
+
+Note that volume requirements (type and/or size) may not be changed after initial deployment.
```

Comment:
**Note:** Volume requirements (type and/or size) cannot be changed after initial deployment.